### ISSUE TITLE
Getting wrong expiration month and year

### ISSUE THREAD

**Issue by neil-119** [triniwiz/nativescript-stripe#13]
I seem to be getting the wrong expMonth and expYear:

    <stripe:CreditCardView #ccview></stripe:CreditCardView>
    ....
    @ViewChild('ccview') ccRef: ElementRef<CreditCardView>;
    
    const card = this.ccRef.nativeElement.card;
    console.log("got card", card.expMonth, card.expYear, card.cvc, card.number);

The CVC and card number show up just fine. But the widget gives e.g. "1053187" and "1053279" when I input 12/22 for example -- this is on Android, haven't tested iOS yet. Any ideas?

**Comment by neil-119** [comment]
Ok, when the card object is instantiated [here](https://github.com/triniwiz/nativescript-stripe/blob/269e39df3163deb52b7f9fb16faa05c27f43680c/src/android/card.ts#L7), it takes `cardExpMonth` (which would be valid, e.g. 12), and then the result of the `new java.lang.Integer(cardExpMonth)` is some large integer -- there is something whacky going on with this conversion. Not sure what it could be -- maybe a bug in NativeScript?

**Comment by neil-119** [comment]
Ok, I figured it out after some debugging. `cardExpMonth` and `cardExpYear` are objects. Calling intValue() before converting it seems to work well. See [my fork](https://github.com/neil-119/nativescript-stripe/blob/b5b10309e4a7f836ccd3e0133533d2e961c23910/src/android/card.ts#L7). As a workaround, I've created an npm package [here](https://www.npmjs.com/package/nativescript-stripe-nr2) that contains the fix. If you use it, you'll have to change your imports until the appropriate fix is determined and merged into this main branch.
